src/Foundation/Enum.cs

```diff
@@ -1258,4 +1259,51 @@ public enum NSMeasurementFormatterUnitOptions : nuint {
 		TemperatureWithoutUnit = (1 << 2)
 	}
 
+	[Watch(2, 0), TV(9, 0), Mac(10, 8), iOS(6, 0)]
+	[Native]
+	public enum NSXPCConnectionOptions : nuint
+	{
+		NSXPCConnectionPrivileged = (1 << 12)
```

Comment:
in this case the enum member name should not be prefixed with `NSXPCConnection` anyway